---
title: "Feature Selection: Genotype Data"
format: pdf
editor: visual
bibliography: references.bib
editor_options:
  markdown:
    wrap: 72
---

Hapmap files were subset to SNPs present in Supplemental Table 6 from
@romeronavarro2017, combined, and converted to VCF using TASSEL version
5.2.64 [@bradbury2007].

```{bash}
# filter 
~/tools/tassel-5-standalone/run_pipeline.pl -Xmx30g -fork1 -h ~/scratch/maize/CIMMYT/GBS/AllZeaGBSv2.7_SEED_Beagle4_chr9.hmp.txt -includeSiteNamesInFile Romero_altitude.txt -export chr9.alt.flt

# combine
~/tools/tassel-5-standalone/run_pipeline.pl -Xmx30g -fork1 -h alt.hmp.txt -fork2 -h alt2.hmp.txt -fork3 -h alt3.hmp.txt -fork4 -h chr10.alt.flt.hmp.txt -combine5 -input1 -input2 -input3 -input4 -mergeGenotypeTables -export romero_alt_zea27 -runfork1 -runfork2 -runfork3 -runfork4

# convert to VCF 
~/tools/tassel-5-standalone/run_pipeline.pl -Xmx30g -fork1 -h romero_alt_zea27.hmp.txt -export romero_alt_zea27 -exportType VCF
```

Coordinates based on the AGPv2 reference genome were then converted to
Zm-B73-REFERENCE-NAM-5.0Zm using the [Assembly Converter online
tool](https://plants.ensembl.org/Zea_mays/Tools/AssemblyConverter?db=core)
from EnsemblPlants, which is based on CrossMap [@zhao2014].

## References
